project.clj

```diff
@@ -1,7 +1,7 @@
 (def ks-version "1.2.0")
 (def tk-version "1.2.0")
 
-(defproject puppetlabs/trapperkeeper-metrics "0.1.3-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-metrics "0.2.0-SNAPSHOT"
```

Comment:
It looks to me like we would merge this PR and then do a minor release from the job https://jenkins-enterprise.delivery.puppetlabs.net/view/clojure-libraries/view/release-jobs/job/enterprise_trapperkeeper-metrics_release-clj_master/build?delay=0sec